book.rb

```diff
@@ -0,0 +1,16 @@
+# book class
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rental_book(rental)
```

Comment:
- [ ] assuming that this method is to add a new rental to the book instance, we could not find the initialisation (with date,  person) of the rental to be added to the book. Let's fix that

Reply:
I'll fix it @EricMbouwe 👍🏼